Jellyfin.Api/Controllers/DynamicHlsController.cs

```diff
@@ -5,6 +5,7 @@
 using System.Globalization;
 using System.IO;
 using System.Linq;
+using System.Runtime.InteropServices;
```

Comment:
What is this import used for?

Reply:
on unix/linux, ffmpeg create fmp4 init file alongside playlist file.
`-hls_fmp4_init_filename "init.mp4"`

but on window, this path needs to be set manually.
`-hls_fmp4_init_filename "X:\transcodes\init.mp4"`